torch/_lobpcg.py

```diff
@@ -939,7 +939,8 @@ def _get_rayleigh_ritz_transform(self, S):
         SBS = _utils.qform(B, S)
         d_row = SBS.diagonal(0, -2, -1) ** -0.5
         d_col = d_row.reshape(d_row.shape[0], 1)
-        R = torch.cholesky((SBS * d_row) * d_col, upper=True)
+        # TODO: could use linalg.cholesky(X, upper=True) once it is implemented
```

Comment:
I don't think adding `upper` argument to torch.linalg.cholesky is in plans.
Better write a "TODO" comment about rewriting this function to return the lower triangular matrix and change the order of multiplications where the result is used.